src/test/java/org/apache/bcel/generic/CountVisitor.java

```diff
@@ -0,0 +1,3118 @@
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.bcel.generic;
+
+import java.util.Objects;
+
+public class CountVisitor implements Visitor {
+    private int aaload;
+    private int aastore;
+    private int aconst_null;
+    private int allocationInstruction;
+    private int aload;
+    private int anewarray;
+    private int areturn;
+    private int arithmeticInstruction;
+    private int arrayInstruction;
+    private int arraylength;
+    private int astore;
+    private int athrow;
+    private int baload;
+    private int bastore;
+    private int bipush;
+    private int branchInstruction;
+    private int BREAKPOINT;
+    private int caload;
+    private int castore;
+    private int checkcast;
+    private int constantPushInstruction;
+    private int conversionInstruction;
+    private int cpInstruction;
+    private int d2F;
+    private int d2I;
+    private int d2L;
+    private int dadd;
+    private int daload;
+    private int dastore;
+    private int dcmpg;
+    private int dcmpl;
+    private int dconst;
+    private int ddiv;
+    private int dload;
+    private int dmul;
+    private int dneg;
+    private int drem;
+    private int dreturn;
+    private int dstore;
+    private int dsub;
+    private int dup;
+    private int dup_x1;
+    private int dup_x2;
+    private int dup2;
+    private int dup2_x1;
+    private int dup2_x2;
+    private int exceptionThrower;
+    private int f2D;
+    private int f2I;
+    private int f2L;
+    private int fadd;
+    private int faload;
+    private int fastore;
+    private int fcmpg;
+    private int fcmpl;
+    private int fconst;
+    private int fdiv;
+    private int fieldInstruction;
+    private int fieldOrMethod;
+    private int fload;
+    private int fmul;
+    private int fneg;
+    private int frem;
+    private int freturn;
+    private int fstore;
+    private int fsub;
+    private int getfield;
+    private int getstatic;
+    private int goTo;
+    private int goTo_w;
+    private int gotoInstruction;
+    private int i2B;
+    private int i2C;
+    private int i2D;
+    private int i2F;
+    private int i2L;
+    private int i2S;
+    private int iadd;
+    private int iaload;
+    private int iand;
+    private int iastore;
+    private int iconst;
+    private int idiv;
+    private int if_acmpeq;
```

Comment:
Use camel case names.

Reply:
Thx, how about variable for `NEW`, `RETURN` and `GOTO`?